web/themes/custom/pineapple/components/_comments-field.twig

```diff
@@ -0,0 +1,45 @@
+{#
+/**
+ * @file
+ * Theme override for comment fields.
+ *
+ * Available variables:
+ * - attributes: HTML attributes for the containing element.
+ * - label_hidden: Whether to show the field label or not.
+ * - title_attributes: HTML attributes for the title.
+ * - label: The label for the field.
+ * - title_prefix: Additional output populated by modules, intended to be
+ *   displayed in front of the main title tag that appears in the template.
+ * - title_suffix: Additional title output populated by modules, intended to
+ *   be displayed after the main title tag that appears in the template.
+ * - comments: List of comments rendered through comment.html.twig.
+ * - comment_form: The 'Add new comment' form.
+ * - comment_display_mode: Is the comments are threaded.
+ * - comment_type: The comment type bundle ID for the comment field.
+ * - entity_type: The entity type to which the field belongs.
+ * - field_name: The name of the field.
+ * - field_type: The type of the field.
+ * - label_display: The display settings for the label.
+ *
+ * @see template_preprocess_field()
+ * @see comment_preprocess_field()
+ */
+#}
+<section class="comments-wrapper">
+  {% if comment_form %}
+    <div class="comment-form-container">
+      <h2 class="subheading">{{ 'Leave a Comment'|t }}</h2>
+      {{ comment_form }}
+    </div>
+  {% endif %}
+
+  {% if comments and not label_hidden %}
+    <h2 class="subheading comments-title">Comments</h2>
```

Comment:
Use heading pattern here. and on line 31.

Reply:
Done